index.html

```diff
@@ -0,0 +1,218 @@
+                   programming skills, engage in interesting and creative projects 
+                   and raise my Front-End development qualifications. Mainly I want to 
+                   focus on learning JavaScript. Also, I put a lot of work to write a 
+                   clear and semantically correct code. Currently, I'm looking for a job 
+                   or internship to improve my skills and knowledge.</p>
+                   <a href="https://docs.google.com/document/d/16re20Kc4HEWw4C594SCu2Kcy7S_gX62trqgDHzsWUdQ/edit?usp=sharing" target="_blank" class="cta">Download Resume</a>
+           </div> 
+        </div>
+    </section>
+    <section id="contact">
+        <div class="contact container">
+            <div><h1 class="section-title">Contact <span>info</span></h1></div>
+                <div class="contact-items">
+                    <div class="contact-item">
+                        <div class="icon"><img src="./img/icons8-phone-100.png" alt="phone_img"></div>
+                        <div class="contact-info">
+                            <h1>Phone</h1>
+                            <a href="tel:+375297676777">+375 29 7676777</a>
+                        </div>
+                    </div>
+                    <div class="contact-item">
+                        <div class="icon"><img src="./img/icons8-email-100.png" alt="email_img"></div>
+                        <div class="contact-info">
+                            <h1>Email</h1>
+                            <a href="mailto:inhuman_@mail.ru">inhuman_@mail.ru</a>
+                         </div>
+                    </div>
+                    <div class="contact-item">
+                        <div class="icon"><img src="./img/icons8-address-100.png" alt="address_img"></div>
+                        <div class="contact-info">
+                            <h1>Address</h1>
+                            <a href="https://www.google.com/maps/@53.9121881,27.6175544,12z" target="_blank">Minsk, Belarus</a>
+                        </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    </main>
+    <footer id="footer">
+        <div class="footer container">
+            <div class="logo"><h1>T<span>y</span>upko Vital<span>y</span></h1></div>
+            <h2>Web solutions</h2>
+            <div class="social-icon">
+                <div class="social-item">
+                    <a href="https://github.com/krilian88" target="_blank"><img src="./img/icons8-github-100.png" alt="github_img"></a>
+                </div>
+                <div class="social-item">
+                    <a href="https://linkedin.com/in/vitali-tyupko-3a33631b7" target="_blank"><img src="./img/icons8-linkedin-100.png" alt="linkedin_img"></a>
+                </div>
+            </div>
+            <p>Copyright © 2021 Minsk</p>
+        </div>
+    </footer>
+    <script src="./app.js"></script>
+  </body>
+</html>
```

Comment:
Please mind the formatting. It is difficult to understand the hierarchy of the elements because of it.